examples/bri-3/src/shared/prisma/prisma.service.ts

```diff
@@ -3,6 +3,9 @@ import { PrismaClient } from '@prisma/client';
 
 @Injectable()
 export class PrismaService extends PrismaClient implements OnModuleInit {
+  constructor() {
+    super({ log: ['info'] });
```

Comment:
why is this needed? default is not info?

Reply:
yes probably not needed
do we need to call super constructor at all?

Reply:
This is how I got it to log when the pool was instantiated. Can leave or take it at this point. Will leave for now.